### DISCUSSION TITLE
Disable Email field Verification? Other workaround to register user without mail?

### DISCUSSION THREAD

**Discussion by PeterEhses** [directus/directus#4360] [Archive]
Hi, I'm trying to use the user model to manage API tokens, for that I need to associate a token with files, items etc., like the user would be, but I won't have emails that I can use as unique identifier. From what I can tell, I could enter anything into the email field for a crude MVP of my setup but there's some error checking im place that rejects anything without [*]@[*].[*] format. 
Would I be able to disable that check or alternatively user a proper non-email identifier field instead? I would be totally fine with leaving it blank and using the raw key for now.

**Comment by philleepflorence** [comment]
How many API users on the average would you create for a single [Directus Project](https://docs.directus.io/guides/projects/)?
What if the username takes the format of an Email Address?

I also use **API Users** and **Tokens** for **Server Middleware to Server API** Connections, where multiple applications connect to a single Directus Project, I use Email Aliases as the unique identifiers but also used for management, communication, and notifications.
